### ISSUE TITLE
fast float: compiler warning about misleading indentation

### ISSUE THREAD

**Issue by pjanx** [mm2/Little-CMS#387]
```
../git/submodules/Little-CMS/plugins/fast_float/src/fast_8_tethra.c: In function ‘PerformanceEval8’:
../git/submodules/Little-CMS/plugins/fast_float/src/fast_8_tethra.c:234:57: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
  234 |                                                         else
      |                                                         ^~~~
../git/submodules/Little-CMS/plugins/fast_float/src/fast_8_tethra.c:246:64: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘else’
  246 |                                                                Rest = c1 * rx + c2 * ry + c3 * rz + 0x8001;
      |                                                                ^~~~
```

(This bothers me mostly because I needed a static build.)

**Comment by mm2** [comment]
Should be fixed by ca207ee4dd4ec490e8a42431784278917ea93ea7

**Comment by pjanx** [comment]
Thanks.
